classes/db/DbQuery.php

```diff
@@ -216,8 +232,11 @@ public function limit($limit, $offset = 0)
 	 */
 	public function build()
 	{
-		$sql = 'SELECT '.((($this->query['select'])) ? implode(",\n", $this->query['select']) : '*')."\n";
-
+		if($this->query['type'] == 'SELECT')
```

Comment:
OOps ! My bad, I forgot a norm ! (And you know that I like it !). I do the modification right now !